stock_valuation_fifo_lot/__manifest__.py

```diff
@@ -9,7 +9,12 @@
     "license": "AGPL-3",
     "author": "Ecosoft, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/stock-logistics-workflow",
-    "depends": ["stock_account", "stock_no_negative", "stock_landed_costs"],
+    "depends": [
+        "stock_account",
+        "stock_no_negative",
+        "stock_landed_costs",
+        "mrp_landed_costs",
```

Comment:
why mandatory install mrp?

Reply:
Absolutely correct, but that comes from the "mrp_landed_costs" module. The dependency on MRP isn't necessary for this module. Several functionalities are being combined in this module that would have to be separated into another module or implemented with a soft dependency.

Reply:
Just FYI, we have split the module to stock_valuation_fifo_lot and stock_valuation_fifo_lot_mrp_landed_cost in [the 16.0 migration PR](https://github.com/OCA/stock-logistics-workflow/pull/1676).